.release/versions.hcl

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
+# This manifest file describes active releases and is consumed by the backport tooling.
+# It is only consumed from the default branch, so backporting changes to this file is not necessary.
+
+schema = 1
+active_versions {
+  version "1.8.x" {
+    ce_active = true
+    lts       = true
+  }
+  version "1.7.x" {
+    ce_active = true
+  }
+  version "1.6.x" {
+    ce_active = true
+  }
+  version "1.5.x" {
+    ce_active = true
+  }
```

Comment:
Just timing-wise, we're in the beta window for 1.8, so I don't think anyone has been backporting to 1.5.x as there's no more planned 1.5.x releases at this point.

Reply:
Noted, I've removed that version from the manifest file.